### DISCUSSION TITLE
[PSD1] Exercise 1.18

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#113] [Esercizi (A-L)]
**Exercise 1.18** Convert the following hexadecimal numbers to decimal. Show your work.

(a) $4E_{16}$
(b) $7C_{16}$
(c) $ED3A_{16}$
(d) $403FB001_{16}$

**Comment by niccolo-fato** [comment]
![photo_5870846455839833120_y](https://user-images.githubusercontent.com/73995334/210087239-9956a014-be7f-432a-be58-ad7847688d62.jpg)
